doc/index.md

```diff
@@ -763,6 +763,8 @@ provides, and a blurb about how each is intended to be used.
   <DD>Given a source range and an integral count, return a range consisting of the first *count* elements from the source range, or the complete range if it has fewer elements. (The result of `view::take` is not a `SizedRange`.)</DD>
 <DT>\link ranges::view::take_exactly_fn `view::take_exactly`\endlink</DT>
   <DD>Given a source range and an integral count, return a range consisting of the first *count* elements from the source range. The source range must have at least that many elements. (The result of `view::take_exactly` is a `SizedRange`.)</DD>
+<DT>\link ranges::view::drop_exactly_fn `view::take_last`\endlink</DT>
```

Comment:
There's a `drop` here that should be changed to `take`.

Reply:
This one's right - there's not actually a `take_last_fn` in this implementation, `take_last` just calls `drop_exactly`. Unless it should like somewhere else?

Reply:
The `_fn` are the range adaptors, which must exist, unlike the `_view`.

Reply:
Oh, I misunderstood everything.

Reply:
I must correct myself. The range adaptors are the `_fn` wrapped in `view::view<>`.